FacuReartes/react-laba-final-project-team-2
# [SPT2-91] Add pull request template

## Description

Add a pull request readme template file to have better pull requests descriptions.

### Changes

- Added `pull_request_template.md` file.

### Jira

- [SPT2-91](https://solvd-final-project-team-2.atlassian.net/browse/SPT2-91)

### How to Test

1. Work on a branch and create a new pull request.
2. This template should show in any new PR.

### Screenshots (if applicable)


### Checklist


### Additional Notes

Evidence in Jira Ticket.

[SPT2-91]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-91?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
